thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -27,10 +87,11 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
   model(params) {
     const { metricId, anomalyId, sessionId } = params;
 
-    let metricUrn, anomalyUrn, session, anomalyContext, anomalySessions;
+    let metricUrn, metricEntity, anomalyUrn, session, anomalyContext, anomalySessions;
 
     if (metricId) {
       metricUrn = `thirdeye:metric:${metricId}`;
+      metricEntity = fetch(`/rootcause/raw?framework=identity&urns=${metricUrn}`).then(checkStatus).then(res => res[0]).catch(() => undefined);
```

Comment:
we can just pass an anonymous function in catch
`.catch(() => {})` 
This essentially achieves the same thing, but looks better imho